automation_tools/__init__.py

```diff
@@ -1781,38 +1781,28 @@ def upstream_install(admin_password=None, run_katello_installer=True, **kwargs):
         return installer_options
 
 
-def downstream_install(sat_version='7', admin_password=None, run_katello_installer=True):
-    """Task to install Satellite 6
+def downstream_install(sat_version='7', admin_password='changeme', run_katello_installer=True,
+                       url=None, maintain_url=None):
+    """Task to install Satellite from internal repo
 
-    The following environment variables affect this command:
-
-    ADMIN_PASSWORD
-        Optional, defaults to 'changeme'. Foreman admin password.
-    BASE_URL
-        URL for the Satellite compose repository.
-    MAINTAIN_REPO
-        URL for the satellite maintenance compose repository
-
-    :param sat_version: Indicates which satellite version is being installed
+    :param str sat_version: Indicates which Satellite version is being installed
+    :param str admin_password: Foreman admin password. Default: 'changeme'.
+    :param bool run_katello_installer: Should the task run installer or just pass installer options
+    :param str url: the repo URL where to install from
+    :param str maintain_url: the maintain repo URL where to install from
 
     """
-    if admin_password is None:
-        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
-
-    base_url = os.environ.get('BASE_URL')
-    if base_url is None:
-        print('The BASE_URL environment variable should be defined')
+    if not url:
+        print('The url parameter should be defined')
```

Comment:
Instead, raise some exception(probably ValueError here) so we need not to `sys.exit(1)` explicitly.

Reply:
Applicable everywhere in PR.

Reply:
Thanks.
Fixed. I also changed text of the messages from suggestion mode to error mode.